pages/workspace/inspect-panel-page.js

```diff
@@ -8,7 +8,7 @@ exports.InspectPanelPage = class InspectPanelPage extends BasePage {
   constructor(page) {
     super(page);
 
-    this.inspectTab = page.locator('div[data-id="inspect"]');
+    this.inspectTab = page.locator('button[data-id="inspect"]');
```

Comment:
I would do the same change commented before as it also has the same "tab" role
![image](https://github.com/user-attachments/assets/881485d8-00b7-462d-907b-0e840ea5484b)
`    this.inspectTab = page.getByRole("tab", { name: "inspect" });
`
Note: I haven't tried this one!